_episodes/06-memory.md

```diff
@@ -25,7 +25,7 @@ There are two reasons that we sub-divide the data into mini-batches during train
 * To produce a better loss curve;
 * To make sure that the data fit into GPU memory.
 
- *"Friends don't let friends use mini-batches larger than 32"* - Yann LeCunn
+ &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;  &nbsp; &nbsp;  &nbsp; &nbsp;&nbsp; &nbsp; *"Friends don't let friends use mini-batches larger than 32"* - Yann LeCunn
```

Comment:
Isn't there a quotation markup one could use? Like simply starting this with `>`? (probably best to be tried out locally)